README.rst

```diff
@@ -16,9 +16,12 @@ CKAN: The Open Source Data Portal Software
     :target: https://circleci.com/gh/ckan/ckan
     :alt: Build Status
 
-.. image:: https://coveralls.io/repos/github/ckan/ckan/badge.svg?branch=master
-    :target: https://coveralls.io/github/ckan/ckan?branch=master
-    :alt: Coverage Status
+.. image:: https://github.com/ckan/ckan/actions/workflows/test.yml/badge.svg
```

Comment:
key to dev-v2.11 branch for this readme
https://github.com/ckan/ckan/actions/workflows/test.yml/badge.svg?branch=dev-v2.11